package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "rental-service",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start": "node dist/index.js",
+    "build": "npm run build-ts",
+    "build-ts": "tsc"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "amqplib": "^0.5.2",
+    "async-retry": "^1.2.1",
+    "body-parser": "^1.18.2",
+    "dotenv": "^5.0.1",
+    "express": "^4.16.3",
+    "helmet": "^3.12.1",
+    "inversify": "^4.13.0",
+    "inversify-express-utils": "^6.0.0",
+    "mongoose": "^5.1.2",
+    "objectmodel": "^3.4.5",
+    "reflect-metadata": "^0.1.12"
+  },
+  "devDependencies": {
+    "@types/amqplib": "^0.5.7",
+    "@types/async-retry": "^1.2.0",
+    "@types/dotenv": "^4.0.3",
+    "@types/express": "^4.11.1",
+    "@types/helmet": "^0.0.37",
+    "@types/mocha": "^5.2.0",
+    "@types/mongoose": "^5.0.11",
+    "@types/node": "^10.0.2",
+    "mocha": "^5.1.1",
```

Comment:
Do you use Mocha to test? So no, remove this devDependency please

Reply:
fixed